packages/worker/src/worker.ts

```diff
@@ -1479,7 +1489,7 @@ export class Worker {
   protected workflow$(): Observable<void> {
     // This Worker did not register any workflows, return early
     if (this.workflowCreator === undefined) {
-      this.log.warn('No workflows registered, not polling for workflow tasks');
+      this.logger.warn('No workflows registered, not polling for workflow tasks');
       this.workflowPollerStateSubject.next('SHUTDOWN');
```

Comment:
logAttributes? There are a few others in this file with no attributes.

Reply:
This is an Workflow Worker lifecycle event, not specific to a single workflow. Hence, no pertinent `logAttributes` to report.